### PR TITLE
Update entrypoint.py

### PR DESCRIPTION
**Pull request by DeepeshHomex**

### CODE REVIEW

File: .github/actions/process-review-comments/entrypoint.py [hunk 1]
```diff
@@ -23,7 +23,8 @@ def review_comment_check(comment_body):
 def review_comment_reply(pull_request_review_id):
     url = "https://api.github.com/repos/{}/pulls/{}/comments/{}/replies".format(repository_name, pr, id)
     print(url)
-    headers = { 'Accept': 'application/vnd.github.v3+json' }
```
Comment (deepeshp656) [inline .github/actions/process-review-comments/entrypoint.py:26]: ⚠️  review etiquette not followed on : ⚠️  review etiquette not followed on : ⚠️  review etiquette not followed on : following review comment does not follow review etiquettewill this https://github.com/HomeXLabs/reviewington/blob/main/docs/pr_etiquette.md  
 
 
 For more information please visit : https://github.com/HomeXLabs/reviewington/blob/main/docs/pr_etiquette.md

Comment (DeepeshHomex) [reply .github/actions/process-review-comments/entrypoint.py:26]: done CHECK.

Comment (DeepeshHomex) [reply .github/actions/process-review-comments/entrypoint.py:26]: check now NNN

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: pmj

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: ihlik

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: test30

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: test31

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: nkkl

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: m,n

Comment (deepeshp656) [reply .github/actions/process-review-comments/entrypoint.py:26]: why

File: .github/actions/process-review-comments/entrypoint.py [hunk 2]
```diff
@@ -13,53 +13,69 @@ def setup_args():
 
     return parser.parse_args()
 
+
 def review_comment_check(comment_body):
-    print(comment_body)
-    res = comment_body.startswith(tuple(pref_list))
```
Comment (DeepeshHomex) [inline .github/actions/process-review-comments/entrypoint.py:18]: ⚠️  review etiquette not followed on : whyy 
 
 For more information please visit : https://github.com/HomeXLabs/reviewington/blob/main/docs/pr_etiquette.md

Comment (DeepeshHomex) [reply .github/actions/process-review-comments/entrypoint.py:18]: ???

**Pull request merged**